src/utils/VerificationValidationParser.cpp

```diff
@@ -186,20 +189,14 @@ Result* Parser::gqa(const QString& cmd, const QString* terminalOutput) {
 }
 
 void Parser::gqaSpecificTest(Result* r, const QString& currentLine, const Test* type) {
-    if (currentLine.trimmed().startsWith("Trying initial grid spacing", Qt::CaseInsensitive) == true || 
-        currentLine.trimmed().startsWith("Using grid spacing lower limit", Qt::CaseInsensitive) == true ||
-        currentLine.trimmed().startsWith("Processing with grid spacing", Qt::CaseInsensitive) == true ||
-        currentLine.trimmed().startsWith("overlap tolerance to", Qt::CaseInsensitive) == true ||
-        currentLine.trimmed().startsWith("NOTE: Stopped, grid spacing refined to", Qt::CaseInsensitive) == true ||
-        currentLine.trimmed().startsWith("Trying initial grid spacing", Qt::CaseInsensitive) == true ||
-        currentLine.trimmed().startsWith("Summary", Qt::CaseInsensitive) == true ||
-        currentLine.trimmed().startsWith("list Overlaps", Qt::CaseInsensitive) == true ||
-        currentLine.trimmed().isEmpty()) return;
+    if (currentLine.trimmed().isEmpty()) return;
     
     if(type == &DefaultTests::NO_NULL_REGIONS)
     {
         QString objectPath1 = currentLine.split(' ')[0];
-        QString objectName1 = objectPath1.mid(objectPath1.lastIndexOf('/'), objectPath1.size() - objectPath1.lastIndexOf('/'));
+        QString objectName1 = objectPath1;
+        if(objectPath1.indexOf('/') != -1)
```

Comment:
I know I'm being really nit-picky here, but:
- could we save result of .indexOf() into a variable so we don't repeatedly call it
- so like "if(resultIdx != -1) { objectName1 = objectPath1.mid(resultIdx, objectPath1.size() - resultIdx); }"